Deepak_Yadav/P1 Profile/css/style.css

```diff
@@ -0,0 +1,47 @@
+.nav1{
+	padding-left: 100px;
+}
+.background-front{
+    background: url(../images/background-front.jpg) no-repeat;
+    background-size: cover;
+    -webkit-filter: blur(0.5px);
+    min-height: 630px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-wrap: wrap;
```

Comment:
use flex-flow instead, its short hand property for flex-direction and flex-wrap

Reply:
ohkk nikhil..i am updating this right now..